src/providers/cross-chain-sync.provider.ts

```diff
@@ -272,7 +273,15 @@ export const crossChainSyncProvider = () => {
   watch(
     () => networksBySyncState.value,
     newVal => {
-      clearTempSyncingNetworks();
+      if (newVal.synced.length > 0) {
+        const hasSyncingMismatch = newVal.syncing.some(network => {
+          return newVal.synced.includes(network);
+        });
+
+        if (hasSyncingMismatch) {
+          clearTempSyncingNetworks();
```

Comment:
The name of this function makes it seem like its emptying the list of temp syncing networks, when it's only removing the ones that are synced. I think it'd be more clear calling it `clearSyncedNetworksFromTempSyncing`